packages/wallets/src/features/cashier/modules/Transactions/components/TransactionsPendingRow/TransactionsPendingRow.tsx

```diff
@@ -39,109 +68,139 @@ const TransactionsCryptoRow: React.FC<TProps> = ({ transaction }) => {
-            </div>
-            <div
-                className={classNames(
-                    'wallets-transactions-pending-row__column',
-                    'wallets-transactions-pending-row__transaction-time'
+                <TransactionsPendingRowField
+                    className={{ 'wallets-transactions-pending-row__transaction-time': !isMobile }}
+                    name='Time'
+                    value={moment
+                        .unix(transaction.submit_date)
+                        .format(isMobile ? 'HH:mm:ss [GMT]' : 'DD MMM YYYY HH:mm:ss [GMT]')}
+                    valueTextProps={{
+                        color: 'general',
+                        size: isMobile ? 'xs' : '2xs',
+                        weight: isMobile ? 'bold' : 'regular',
+                    }}
+                />
+                {!isMobile && (
+                    <div className='wallets-transactions-pending-row__transaction-amount'>
+                        <WalletText
+                            align='right'
+                            color={transaction.is_deposit ? 'success' : 'red'}
+                            size='sm'
+                            weight='bold'
+                        >
+                            {transaction.is_deposit ? '+' : '-'}
+                            {transaction.formatted_amount}
+                        </WalletText>
+                    </div>
                 )}
-            >
-                <WalletText color='primary' size='xs'>
-                    Time
-                </WalletText>
-                <WalletText color='general' size='2xs'>
-                    {moment.unix(transaction.submit_date).format('DD MMM YYYY HH:mm:ss [GMT]')}
-                </WalletText>
             </div>
-            <div
-                className={classNames(
-                    'wallets-transactions-pending-row__column',
-                    'wallets-transactions-pending-row__transaction-amount'
-                )}
+            <button
+                className='wallets-transactions-pending-row__transaction-status'
+                onClick={
+                    isMobile
+                        ? () =>
+                              modal.show(
+                                  <WalletActionModal
+                                      actionButtonsOptions={[
+                                          {
+                                              isPrimary: true,
+                                              onClick: modal.hide,
+                                              text: 'Ok',
+                                          },
+                                      ]}
+                                      description={transaction.description}
+                                      hideCloseButton
+                                      title='Transaction details'
+                                  />,
+                                  { defaultRootId: 'wallets_modal_root' }
```

Comment:
Couldn't this be a function, like `onTransactionDetailsClick` or something like that?

Reply:
Done